### PR TITLE
Add Hotbar feature page

### PR DESCRIPTION
**Pull request by SpectralCathe**

### CODE REVIEW

File: docs/Features/img/hotbar/Thumbs.db
Comment (SpectralCathe) [inline docs/Features/img/hotbar/Thumbs.db:1]: who put this here

File: docs/Features/Hotbar.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Action Bar
+
+The buttons next to the portrait are now customizable; when your hotbar is unlocked, you can drag and drop to rearrange them. Additionally, mods can add their own "actions" there, and there are a few new built-in ones. You can press the middle mouse button to unbind them.  
+
+Right click one of those buttons or click the `^` button to bring up a "drawer" with all the actions that you can drag and drop onto your actions bar. You can also click them to use them directly from there.  
+
+![Hotbar action drawer preview.](img/hotbar/drawer.png)
+
+**Action bar has to be unlocked to allow drag and drop! The lock button is located right under the +/- buttons!**
```
Comment (PinewoodPip) [inline docs/Features/Hotbar.md:38]: Exclamation mark at the end here is unnecessary.

File: docs/Features/Hotbar.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Existing groups can be dragged around by their handle. Right-clicking a Skill Group will bring up a submenu with *Delete* and *Resize* options.  
+
+## Action Bar
+
+The buttons next to the portrait are now customizable; when your hotbar is unlocked, you can drag and drop to rearrange them. Additionally, mods can add their own "actions" there, and there are a few new built-in ones. You can press the middle mouse button to unbind them.  
+
+Right click one of those buttons or click the `^` button to bring up a "drawer" with all the actions that you can drag and drop onto your actions bar. You can also click them to use them directly from there.  
+
+![Hotbar action drawer preview.](img/hotbar/drawer.png)
```
Comment (PinewoodPip) [inline docs/Features/Hotbar.md:36]: Screenshot should be taken with the hotkey bindings enabled.

File: docs/Features/Hotbar.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+You can disable this in **Options -> Epip Settings -> Hotbar -> Hotbar Buttons Area Sizing**.  
+
+These buttons can be used with keyboard hotkeys; to do so, you must assign keys to them in **Options -> Controls -> Epip Encounters -> Hotbar Action #N (top/bottom row)**.  
+You can toggle whether assigned hotkeys are displayed on the buttons in **Options -> Epip Settings -> Hotbar -> Show Action Hotkeys**.
+
+### New Actions
+
+Epip Encounters implements multiple handy new actions, in addition to all vanilla functions:
+
+- ![Hotbar action pyramids](img/hotbar/action_pyramids.png) - Pyramids: Uses the teleporter pyramid on current character.
```
Comment (PinewoodPip) [inline docs/Features/Hotbar.md:50]: The "-" looks pretty bad: 
image

Comment (PinewoodPip) [inline docs/Features/Hotbar.md:50]: Confusing wording, it uses the TP pyramids from anywhere in the party inventory

File: docs/Features/Hotbar.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+- ![Hotbar action chain](img/hotbar/action_chain.png) - Chain/Unchain: Toggle chaining with your nearby party members. Instant, plays no animation.
+- ![Hotbar action rest](img/hotbar/action_rest.png) - Bedroll Rest: Requires **any** of your party members to have a bedroll in inventory.
+- ![Hotbar action vanity](img/hotbar/action_vanity.png) - Vanity: A Vanity menu shortcut.
```
Comment (PinewoodPip) [inline docs/Features/Hotbar.md:53]: Would reword to "Opens the Vanity menu". Also, link to the vanity page

File: docs/Features/Hotbar.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+These buttons can be used with keyboard hotkeys; to do so, you must assign keys to them in **Options -> Controls -> Epip Encounters -> Hotbar Action #N (top/bottom row)**.  
+You can toggle whether assigned hotkeys are displayed on the buttons in **Options -> Epip Settings -> Hotbar -> Show Action Hotkeys**.
+
+### New Actions
+
+Epip Encounters implements multiple handy new actions, in addition to all vanilla functions:
+
+- ![Hotbar action pyramids](img/hotbar/action_pyramids.png) - Pyramids: Uses the teleporter pyramid on current character.
+- ![Hotbar action chain](img/hotbar/action_chain.png) - Chain/Unchain: Toggle chaining with your nearby party members. Instant, plays no animation.
+- ![Hotbar action rest](img/hotbar/action_rest.png) - Bedroll Rest: Requires **any** of your party members to have a bedroll in inventory.
```
Comment (PinewoodPip) [inline docs/Features/Hotbar.md:52]: Would reword to "Uses a bedroll from the party inventory."
The bullet points should explain what the buttons do, even if to you it seems obvious. In this case the requirement (using my wording) is implicit.

File: docs/Features/Hotbar.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+- ![Hotbar action vanity](img/hotbar/action_vanity.png) - Vanity: A Vanity menu shortcut.
+- ![Hotbar action respec](img/hotbar/action_respec.png) - Respec: Requires **any** of your party members to have a Portable Respec Mirror provided by [this mod](https://steamcommunity.com/sharedfiles/filedetails/?id=1507233042) in inventory.
+- ![Hotbar action meditate](img/hotbar/action_meditate.png) - Meditate
```
Comment (PinewoodPip) [inline docs/Features/Hotbar.md:55]: Again, would reword to be clear on what these do. "Opens the Mediate UI".

File: docs/Features/Hotbar.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+- ![Hotbar action SI](img/hotbar/action_si.png) - Source Infuse
+
+The purpose of the action bar is to provide a centralized place for modders to add shortcuts/hotkeys to their interfaces or features, eliminating the need for interactable books that clutter your inventory and freeing up space on the hotbar.
+
+[The hotbar improvements are also available as a standalone mod with no EE dependency.](https://steamcommunity.com/sharedfiles/filedetails/?id=2759281297)
```
Comment (PinewoodPip) [inline docs/Features/Hotbar.md:60]: The hotbar mod is now redundant and should no longer be mentioned.

**Review by PinewoodPip — CHANGES_REQUESTED** [review]
